src/modules/billing/controllers/invoices.js

```diff
@@ -14,4 +14,25 @@ const patchInvoice = async request => {
   }
 };
 
+const resetIsFlaggedForRebilling = async request => {
```

Comment:
Sorry if I didn't explain this better after my first review. The resetting of the rebilling status for the original invoice id is already handled on line 115 in the delete invoice controller so you don't need these controller functions here and you don;t need the routes for these either.